src/token/types.ts

```diff
@@ -13,9 +13,32 @@ export type Token = {
   symbol: string; // A ticker symbol or shorthand, up to 11 characters
 };
 
+// The raw response from the Swap API
+export type RawTokenData = {
```

Comment:
`RawTokenData` I feel like this can be a bit more explicit as name.

Reply:
also should be alphabetic the type.

Reply:
renamed to `LegacyTokenData`

Reply:
and sorted

Reply:
dope